test/e2e/tools/event-writer/packages.config

```diff
@@ -0,0 +1,10 @@
+﻿﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="eBPF-for-Windows" version="0.19.2" targetFramework="native" />
```

Comment:
This shouldn't be needed since the platform-specific variants are included below.